src/mediumComplexityProblem6/ReverseString.java

```diff
@@ -0,0 +1,31 @@
+package mediumComplexityProblem6;
+
+public class ReverseString {
+    public void reverseStringOperation(char[] inputCharArray) {
+        int startCounter = 0;
+        int endCounter = inputCharArray.length - 1;
+        while (startCounter < endCounter) {
+            if (!Character.isAlphabetic(inputCharArray[startCounter])) {
```

Comment:
check for the numeric character also, try to consider edge cases also

Reply:
Done!!